README.md

```diff
@@ -29,25 +29,25 @@ points to the development branch of Zephyr, also `main`.
 [board_porting]: https://docs.zephyrproject.org/latest/guides/porting/board_porting.html
 [bindings]: https://docs.zephyrproject.org/latest/guides/dts/bindings.html
 [drivers]: https://docs.zephyrproject.org/latest/reference/drivers/index.html
-[zephyr]: https://github.com/zephyrproject-rtos/zephyr
+[sdk-nrf]: https://github.com/nrfconnect/sdk-nrf
 [west_ext]: https://docs.zephyrproject.org/latest/develop/west/extensions.html
 
-## Getting Started
+## Getting started
 
-Before getting started, make sure you have a proper Zephyr development
-environment. Follow the official
-[Zephyr Getting Started Guide](https://docs.zephyrproject.org/latest/getting_started/index.html).
+Before getting started, make sure you have a proper nRF Connect SDK development environment.
+Follow the official
+[Getting started guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started.html).
```

Comment:
```suggestion
[Installation guide](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/getting_started.html).
```
We do not have Getting started anymore. We have a redirect, but you could use https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation.html, instead.